portal/templates/initial_queries_macros.html

```diff
@@ -112,16 +112,26 @@ <h4 class="tnth-headline">{{ _("Terms of Use") }}</h4>
           <br />
           <p>{{ _("Have you had a prostate cancer biopsy?") }}</p>
           <div class="form-group">
-            <div class="radio">
+            <div class="radio biopsy-option">
               <label>
                 <input type="radio" class="init-queries-field" name="biopsy" id="biopsy_yes" value="true"> {{ _("Yes") }}
               </label>
             </div>
-            <div class="radio">
+            <div id="biopsyDateContainer" class="tnth-hide indent form-group">
+            	<label for="biopsyDate">{{_("Biopsy Date: ")}}</label>
```

Comment:
You might want to move the colon outside the translation call (ie `{{_("Biopsy Date")}}:`), that way you can reuse that translation other places in the UI without having to add translations with and without a colon.